charts/flowise/templates/deployment.yaml

```diff
@@ -86,7 +86,7 @@ spec:
             - /bin/sh
             - -ec
             - |
-              wait4x postgresql "postgres://${POSTGRESQL_USERNAME}:${POSTGRESQL_PASSWORD}@${POSTGRESQL_HOST}:${POSTGRESQL_PORT}/${POSTGRESQL_DATABASE}?sslmode=disable" --timeout 0
+              wait4x postgresql "postgres://${POSTGRESQL_USERNAME}:${POSTGRESQL_PASSWORD}@${POSTGRESQL_HOST}:${POSTGRESQL_PORT}/${POSTGRESQL_DATABASE}?sslmode=${POSTGRESQL_SSL}" --timeout 0
```

Comment:
As per https://www.postgresql.org/docs/current/libpq-ssl.html I'm not sure a simple true/false will work here?